[2022-09-05 03:07:52,287] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-26T00:00:00+00:00 [queued]>
[2022-09-05 03:07:52,464] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-26T00:00:00+00:00 [queued]>
[2022-09-05 03:07:52,465] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:52,480] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2022-09-05 03:07:52,480] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:52,581] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-08-26T00:00:00+00:00
[2022-09-05 03:07:52,638] {standard_task_runner.py:52} INFO - Started process 439 to run task
[2022-09-05 03:07:52,699] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2022-08-26T00:00:00+00:00', '--job-id', '86', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpt8vyavj9', '--error-file', '/tmp/tmpw0r61n6x']
[2022-09-05 03:07:52,711] {standard_task_runner.py:77} INFO - Job 86: Subtask second_task
[2022-09-05 03:07:53,087] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2022-08-26T00:00:00+00:00 [running]> on host 2b8e628fc16c
[2022-09-05 03:07:53,390] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-26T00:00:00+00:00
[2022-09-05 03:07:53,391] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-09-05 03:07:53,394] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2022-09-05 03:07:53,455] {bash.py:169} INFO - Output:
[2022-09-05 03:07:53,476] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2022-09-05 03:07:53,479] {bash.py:177} INFO - Command exited with return code 0
[2022-09-05 03:07:53,682] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20220826T000000, start_date=20220905T030752, end_date=20220905T030753
[2022-09-05 03:07:54,266] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-05 03:07:54,296] {local_task_job.py:146} INFO - Task exited with return code 0
